cves/kernel/CVE-2021-27365.yml

```diff
@@ -526,9 +567,7 @@ CWE_instructions: |
     CWE: 123            # also ok
 CWE:
 - '787'
-CWE_note: |
-  CWE as registered in the NVD. If you are curating, check that this
-  is correct and replace this comment with "Manually confirmed".
+CWE_note: Manually Confirmed
 nickname_instructions: |
```

Comment:
In terms of a potential nickname, you can tag this CVE with something along the lines of lack of size checking